runtime/src/snapshot_utils.rs

```diff
@@ -59,6 +60,7 @@ pub const INCREMENTAL_SNAPSHOT_ARCHIVE_FILENAME_REGEX: &str = r"^incremental-sna
 
 #[derive(Copy, Clone, Eq, PartialEq, Debug)]
 pub enum SnapshotVersion {
+    V1_2_1,
     V1_2_0,
 }
```

Comment:
This new snapshot version should be 1.3 instead of 1.2.1.

I was doing 1.2.1 because I wanted to remove some unused fields from the snapshot but hadn't changed anything else, functionality-wise (IIRC). Since this PR would be changing/adding functionality, v1.3 I think is the right-er call. Note that the version numbers here are somewhat arbitrary and do _not_ follow [Semantic Versioning](https://semver.org/).

Reply:
Sure thing.  I will change it to V1.3.0